frontend/src/components/Tabcordian.tsx

```diff
@@ -0,0 +1,107 @@
+import Link from "next/link"
+import { FC, useState } from "react"
+import { MdKeyboardArrowDown, MdKeyboardArrowRight } from 'react-icons/md'
+
+interface linkArray {
```

Comment:
The interface name should be `TabcordianLink` and the case should be PascalCase, imo.

Reply:
renamed